leaflet-treering.js

```diff
@@ -127,16 +127,19 @@ function LTreering (viewer, basePath, options) {
       this.ioTools.bar.addTo(this.viewer);
       this.settings.bar.addTo(this.viewer);
       this.undoRedoBar.addTo(this.viewer);
+      //overlay 'points' option enabled by default in popout
+      L.control.layers(this.baseLayer, this.overlay).addTo(this.viewer);
     } else {
       this.popout.btn.addTo(this.viewer);
       this.viewData.btn.addTo(this.viewer);
       this.ioTools.bar.addTo(this.viewer);
       this.settings.bar.addTo(this.viewer);
+      L.control.layers(this.baseLayer, this.overlay).addTo(this.viewer);
+      //defaults overlay 'points' option to disabled
```

Comment:
I'd avoid duplicating code like this - just makes things more fragile. If you move the whole "L.control.layers(this.baseLayer, this.overlay).addTo(this.viewer);" line up above the if/else block, you only need one copy of it. Then the only thing you need inside the else{} is the map.removeLayer.